### PR TITLE
added navigation links to website

### PR DESCRIPTION
**Pull request by awelllle**
#75 
I left out the about us link for now until the new about the team page is built. I think I'll look into it

### CODE REVIEW

File: _includes/scripts.html [hunk 1]
```diff
@@ -25,4 +25,34 @@
     });
   });
 </script>
+
+
+<script>
```
Comment (RichardLitt) [inline _includes/scripts.html]: No need for an extra script tag - just put it in the older one.

File: _includes/navigation.html [hunk 2]
```diff
@@ -11,8 +11,37 @@
...
-</div><!-- /.navigation-wrapper -->
+</div><! /.navigation-wrapper --
+<div class="js-menu-screen menu-screen"></div> -->
 
-{% include browser-upgrade.html %}
 
-<div class="js-menu-screen menu-screen"></div>
+<nav class="main-navigation " id="navbar">
+
+	<!-- <div class="navbar-header animated fadeInUp">
```
Comment (RichardLitt) [inline _includes/navigation.html]: Remove dead comments.

File: _includes/navigation.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div class="navigation-wrapper">
+<!-- <div class="navigation-wrapper">
```
Comment (RichardLitt) [inline _includes/navigation.html:1]: Just delete this block, instead of commenting it out.

File: _includes/head.html [hunk 4]
```diff
@@ -23,7 +23,7 @@
 
 <!-- For all browsers -->
 <link rel="stylesheet" href="{{ BASE_URL }}/assets/css/main.css">
-
+<link rel="stylesheet" href="{{ BASE_URL }}/assets/css/custom.css">
```
Comment (RichardLitt) [inline _includes/head.html:26]: Let's use Sass! I would create a file called `_nav.scss` in the scss folder, and then import it there. It's a lot more powerful than CSS, and makes things more streamlined. That's why you couldn't find the CSS folder - that's where I've been editing stuff.

File: _includes/scripts.html [hunk 5]
```diff
@@ -25,4 +25,34 @@
...
+  $(document).ready(function(){
+    // Add smooth scrolling to all links
+    $(".nav-link").on('click', function(event) {
```
Comment (RichardLitt) [inline _includes/scripts.html]: You've made `.nav-link` both a class for functionality, and a style. Could you make two classes, instead? We need to add this slow-scrolling functionality to the links in the footer, too.

**Review by RichardLitt — APPROVED** [review]
Very good work, @awelllle! Thank you. 

I've added some comments. Let me know if you want help with any of these.

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Sorry, meant to press 'Request Changes'.

File: _includes/scripts.html (hunk 5 above)
Comment (awelllle) [reply _includes/scripts.html]: so maybe it could be named 'slow-scroll' or something? so that it can be applied to the footer too?

Comment (RichardLitt) [reply _includes/scripts.html]: That works!

File: _includes/head.html (hunk 4 above)
Comment (awelllle) [reply _includes/head.html:26]: I think I need a little bit more clarification on this. I understand the part about creating '_nav.scss' in the '_sass' folder.

Are you saying I should create another file called 'custom.css' in the '_sass' folder? 

Then do something like '@import "custom.css" inside '_nav.scss' ?

Comment (RichardLitt) [reply _includes/head.html:26]: Don't create a `.css` file at all. Jekyll will automatically create it for you. import `_nav` in `main.scss`. :)

**Comment by awelllle** [comment]
Checks are failing. Is there something I'm not doing?

**Comment by RichardLitt** [comment]
Fixed and merged! You were pretty close. Take a look at my changes in 423ddb1c329798c57ef2707c3e527fd746f68c3e. :) Thank you, @awelllle. This is great.

**Pull request merged**